internal/project/session.go

```diff
@@ -190,40 +193,52 @@ func (s *Session) GetCurrentDirectory() string {
 	return s.options.CurrentDirectory
 }
 
-// Gets current UserPreferences, always a copy
+// Gets copy of current configuration
+func (s *Session) Config() *Config {
+	s.configRWMu.Lock()
+	defer s.configRWMu.Unlock()
+	return s.workspaceConfig.Copy()
+}
+
+// !!! ts/js preferences
+// Gets copy of current UserPreferences
 func (s *Session) UserPreferences() *lsutil.UserPreferences {
 	s.configRWMu.Lock()
 	defer s.configRWMu.Unlock()
-	return s.userPreferences.Copy()
+	return s.workspaceConfig.Ts.Copy()
```

Comment:
What is the plan for returning the "right" one?

If we aren't handling that yet, can we just skip doing the JS stuff for the time being?

Reply:
Added JS handling